Cows-HacknSlash/Assets/Scripts/Inventory script/Inventory.cs

```diff
@@ -7,23 +7,61 @@ public class Inventory : MonoBehaviour {
     // Item array with all items able to go into the inventory.
     public Item[] Alltheitems;
 
+    public static string itemselected;
     // The actual item in the inventory, later being set to the item of choice.
     public GameObject itemdisplayerprefab;
 
     public GameObject[] Slots;
 
     public GameObject Backpack;
 
+    public GameObject Slotselector;
+
+    public int slotselected;
+
+    public TextMeshProUGUI slotinfo;
+
     private void Awake()
     {
         Backpack.SetActive(false);
     }
 
     private void Update()
     {
+        if (Input.GetKeyDown(KeyCode.Alpha1))
+        {
+            Slotselector.transform.position = Slots[0].transform.position;
+            slotselected = 0;
+        }
+        if (Input.GetKeyDown(KeyCode.Alpha2))
+        {
+            Slotselector.transform.position = Slots[1].transform.position;
+            slotselected = 1;
+        }
+        if (Input.GetKeyDown(KeyCode.Alpha3))
+        {
+            Slotselector.transform.position = Slots[2].transform.position;
+            slotselected = 2;
+        }
+        if (Input.GetKeyDown(KeyCode.Alpha4))
+        {
+            Slotselector.transform.position = Slots[3].transform.position;
+            slotselected = 3;
+        }
+        if (Input.GetKeyDown(KeyCode.Alpha5))
+        {
+            Slotselector.transform.position = Slots[4].transform.position;
+            slotselected = 4;
+        }
+
+        //Set the info bar too what item selected name is
```

Comment:
Format the comments like this: `//<space><Comment capitalized><period>` e.g. `// Whassup.`